libraries/AP_Proximity/AP_Proximity_Backend.cpp

```diff
@@ -254,19 +147,17 @@ void AP_Proximity_Backend::database_push(float angle, float distance)
 }
 
 // update Object Avoidance database with Earth-frame point
-void AP_Proximity_Backend::database_push(float angle, float distance, uint32_t timestamp_ms, const Vector3f &current_pos, const Matrix3f &body_to_ned)
+// pitch can be optionally provided if needed
+void AP_Proximity_Backend::database_push(float angle, float distance, uint32_t timestamp_ms, const Vector3f &current_pos, const Matrix3f &body_to_ned, float pitch)
```

Comment:
the "pitch" argument should probably be moved to be between "angle" and "distance"

Reply:
It is done this way to get the default value of zero. However I can add another method seperately which takes in the pitch value. Would you rather want that?